test/functional/android/appium_tests.py

```diff
@@ -98,32 +101,39 @@ def test_current_package(self):
         package = self.driver.current_package
         self.assertEqual('com.example.android.apis', package)
 
-    def test_pull_file(self):
-        data = self.driver.pull_file('data/local/tmp/strings.json')
-        strings = json.loads(data.decode('base64', 'strict'))
-        self.assertEqual('You can\'t wipe my data, you are a monkey!', strings[u'monkey_wipe_data'])
+    def test_push_pull_file(self):
+        path = '/data/local/tmp/test_push_file.txt'
+        data = b'This is the contents of the file to push to the device.'
+
+        self.driver.push_file(path, base64.b64encode(data).decode('utf-8'))
+        data_ret = base64.b64decode(self.driver.pull_file(path))
+
+        # python2
+        # self.driver.push_file(path, data.encode('base64'))
```

Comment:
Should Python 2 separates here?

`base64.b64encode(b'This is the contents of the file to push to the device.').decode('utf-8')`
=> `u'VGhpcyBpcyB0aGUgY29udGVudHMgb2YgdGhlIGZpbGUgdG8gcHVzaCB0byB0aGUgZGV2aWNlLg=='`

in Python 2

Reply:
Now enabled running on both python2 and python3

```
$python2 appium_tests.py
--
$python3 appium_tests.py
```

Reply:
I misunderstood, I'll check the result also in py2 by its codes.